ci/script.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+source ./tempEnv.sh
+
+set -eEuo pipefail
+
+echo "=== Local variables ==="
+PROD_RELEASE_BRANCH="${PROD_RELEASE_BRANCH:-main}"
+DEV_RELEASE_BRANCH="${DEV_RELEASE_BRANCH:-development}"
+DOCKER_IMAGE_NAME="${DOCKER_IMAGE_NAME:-horizenlabs/blockscout-frontend}"
+
+# absolute path to project from relative location of this script
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
+echo "=== Workdir: ${workdir} ==="
+
+command -v docker &> /dev/null && have_docker="true" || have_docker="false"
+echo "=== Have docker: ${have_docker} ==="
+
+# Functions
+function fn_die() {
+  echo -e "$1" >&2
+  exit "${2:-1}"
+}
+
+echo "=== Checking if Docker is installed ==="
+if [ "${have_docker}" = "false" ]; then
+  fn_die "Docker is not installed. Exiting ..."
+fi
+
+echo "=== Checking if GITHUB_TOKEN is set ==="
+if [ -z "${GITHUB_TOKEN:-}" ]; then
+  fn_die "GITHUB_TOKEN variable is not set. Exiting ..."
+fi
+
+echo "=== Checking if DOCKER_WRITER_PASSWORD is set ==="
+if [ -z "${DOCKER_WRITER_PASSWORD:-}" ]; then
+  fn_die "DOCKER_WRITER_PASSWORD variable is not set. Exiting ..."
+fi
+
+echo "=== Checking if DOCKER_WRITER_USERNAME is set ==="
+if [ -z "${DOCKER_WRITER_USERNAME:-}" ]; then
+  fn_die "DOCKER_WRITER_USERNAME variable is not set. Exiting ..."
+fi
+
+docker_tag=""
+if [ "${IS_A_RELEASE}" = "true" ]; then
+  docker_tag="${TRAVIS_TAG}"
+  # Build both versioned and latest tag on release
+  docker_tags="${TRAVIS_TAG},latest"
+elif [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+  if [ "${TRAVIS_BRANCH}" = "${PROD_RELEASE_BRANCH}" ]; then
+    docker_tag=latest
+  elif [ "${TRAVIS_BRANCH}" = "${DEV_RELEASE_BRANCH}" ]; then
+    docker_tag=dev
+  fi
+  docker_tags="${docker_tag}"
```

Comment:
Hm IF it is NOT a release we will be building `latest` docker image tag and overwriting it on Docker Hub. Is this what we want?

Reply:
That's what I understood from this conversation: https://github.com/HorizenLabs/blockscout-frontend/pull/11#discussion_r1567556111

That we want to generate latest image both on releases and not releases but prod branches. You're right that this would overwrite latest image, but I think that's what we want, to always generate latest image on changes to prod branch, and for releases we use the versioned tag. 

Does this make sense @ysibirski ?

Reply:
Yes it does @jmontesdeocanuez. As long as it is intended then this code will do it :)